src/Core/Content/Media/Exception/ThumbnailNotSupportedException.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Media\Exception;
+
+use Shopware\Core\Framework\ShopwareHttpException;
+use Symfony\Component\HttpFoundation\Response;
+
+class ThumbnailNotSupportedException extends ShopwareHttpException
```

Comment:
What about extending the `FileTypeNotSupportedException` here for backwards compatibility reasons? :thinking: 

that way old code that has a `try { ... } { catch FileTypeNotSupportedException }` around the thumbnail generation still works as before :thinking:

Reply:
Yes, got this point worh soner, too. but then I'll have to call construct of ShopwareHttpException, haven't I?
This feels ugly?

Reply:
Yep but as there are good reasons to catch this specific exception we should keep it compatible until 6.5, even if it's ugly.

What about calling the parent constructor and then overriding the protected properties again?
```php
public function __construct()
{
    parent::__construct();
    $this->parameters = $parameters;
    $this->message = $this->parse($parameters, 'The new message template');
}
```

Of course `ThumbnailNotSupportedException` should be marked as deprecated and hinted that it will only extend the `ShopwareHttpException` from 6.5.0 onwards, then we can remove that hack

Reply:
@keulinho sounds good. after last coffee  I recognized the name `FileTypeNotSupportedException` is not suitable, too.

What do you think of deprecating this class and add `FileExtensionNotSupportedException` instead?

Reply:
yep that would make stuff even easier right? as you can change the constructor of the `old` exception to accept message and params and just pass the to the `ShopwareHttpException` and the two new exceptions can both extend the old one for now, and with 6.5.0 we will remove the `FileTypeNotSupportedException` and the new exception will extend `ShopwareHttpException` directly

Reply:
Ahh, good catch! This is really better! Didn't think of "constructors are non-breaking" 👍